src/script/auth/page/Login.tsx

```diff
@@ -65,8 +65,12 @@ import {parseError, parseValidationErrors} from '../util/errorUtil';
 import {UrlUtil} from '@wireapp/commons';
 import Page from './Page';
 import EntropyContainer from './EntropyContainer';
+import {t} from 'Util/LocalizerUtil';
 
-interface Props extends React.HTMLProps<HTMLDivElement> {}
+interface Props extends React.HTMLProps<HTMLDivElement> {
+  changeTitle: (title: string) => void;
+  title: string;
```

Comment:
i was wondering, since these same props are being added to every page here can we export them as an extra interface from somewhere general and just add it to the extends?

Reply:
yes we can do that. I waned to talk about it.

Reply:
Actually can we try this option as well, I feel like it's going to be much simpler
![image](https://user-images.githubusercontent.com/1090716/174287713-cd524412-43b7-4c24-ae55-d794b23d26ea.png)